Dockerfile

```diff
@@ -1,22 +1,48 @@
-FROM comicrelief/php7-slim:7.1
+FROM crs4/php7-base:7.1
 
 LABEL Description="Metadata backend for the PhenoMeNal Portal"
 LABEL software="PhenoMeNal Portal"
 LABEL software.version="1.1.2"
 LABEL version="0.2.2"
```

Comment:
Given that you are expected to use the container through the `entrypoint.sh`, I would deem that as an "API change", which would mean that you should bump the major version number in the `version` label. Please do so as part of this PR.

Reply:
Ok! Major version has been updated via https://github.com/phnmnl/container-portal-metadata-backend/pull/1/commits/1a904da3d5dc3ec2255436233cd47f292c6185d0